### DISCUSSION TITLE
github copilot chat now cannot see the selected code

### DISCUSSION THREAD

**Discussion by torturado** [community/community#63942] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I remeber some days ago, in another computer, i could select the whole python code and then do ctrl+shift+i and write /simplify simplify the selected code and then github simplified my code, but now says that cannot see the selected code

**Comment by torturado** [comment, marked as answer]
the problem is that the code that im using is too long. not a bug

**Reply by DeVitoC** [reply]
It says that it does not have limitations though. Why would this be an issue? If it does have limitations, it needs to be able to state those limitations.

**Comment by srikanthapanda** [comment]
GNU nano 8.5       snake_bot.py       Modified
      issues: write
    steps:
      - run: |
          gh issue --repo ${{ github.repository }>
            create --title "Issue title" --body ">
        env:
          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}

name: Open new issue
on:
  workflow_dispatch:
jobs:
  open-issue:
    runs-on: ubuntu-latest
    permissions:
      contents: read
      issues: write
    steps:
      - run: |
          gh issue create --repo ${{ github.repos>
            --title "Issue title" \
            --body "Issue body"
        env:
          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
